ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-commands.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# TODO(dviroel): extend build step to build all other openstack-k8s-operators
+ORG="openstack-k8s-operators"
+BASE="openstack-operator"
+
+# We don't want to use OpenShift-CI build cluster namespace
+unset NAMESPACE
+
+# Check org and project from job's spec
+REF_BASE=$(echo ${JOB_SPEC} | jq -r '.refs.repo')
+REF_ORG=$(echo ${JOB_SPEC} | jq -r '.refs.org')
+# PR SHA
+PR_SHA=$(echo ${JOB_SPEC} | jq -r '.refs.pulls[0].sha')
+
+# Fail if this step is being used for testing a repo != openstack-operator
+if [[ "$REF_ORG" != "$ORG" || "$REF_BASE" != "$BASE" ]]; then
```

Comment:
Jobs can be triggered by changes on openshift/release repo too.
And at this moment, we only build openstack-operator images, so we will fail when if someone try to use this step with other operator.

Reply:
Is there a plan to extend this job to have a similar deployment in the other operator repos (cinder, glance, etc) ?
Let's say I make a change in `Glance`: it would be great going through similar steps to build the image and get it deployed by the `openstack-operator` to gate a specific PR.
However, we should probably have additional steps to update the `openstack-operator` CR before going through the deployment, so we have to take care about dependencies.
If something changes in the `openstack-operator` logic, then I guess this change is well enough, but at some point this approach should be evolved to work with some sort of `Depends-on` that can be processed extracting the info from the `commitmsg`, checkout the right branches/PRs and build everything accordingly.
@viroel not strictly related to this change itself (which is awesome in terms of progresses we're making), but something we should follow up.

Reply:
Yes, that is the idea, as our next step. We plan to extend the process of build to also build other operators. The idea is to have those jobs running against their own repos, so a PR on Cinder, would trigger an additional step to build Cinder operator/bundle/index, push to our registry, update references on openstack-operator (e.g. api) and build the openstack-operator. Each repo may have its own tests to run after that, or even their own scenario, but the build and deploy process may be generic. This is the starting point.